README.md

```diff
@@ -0,0 +1,55 @@
+# Charmed Mongos Documentation
+
+[mongos](https://www.mongodb.com/docs/v6.0/reference/program/mongos/) is a popular NoSQL database application. It stores its data with JSON-like documents creating a flexible user experience with easy-to-use data aggregation for data analytics. In addition, it is a distributed database, so vertical and horizontal scaling come naturally.
```

Comment:
should mongos have the same definition as mongodb (i.e terms like: nosql database ... stores ... documents...)?